consensus/src/marshal/mod.rs

```diff
@@ -740,13 +756,15 @@ mod tests {
 
             // Block3: Notarized by the actor
             let proposal3 = Proposal {
-                round: Round::new(0, 3),
-                parent: 2,
+                round: Round::new(Epoch::new(0), View::new(3)),
```

Comment:
worthwhile to have a `Round` method be able to take u64's directly? I see why you made it as safe as possible but this is almost too much boilerplate